docs/DeveloperGuide.adoc

```diff
@@ -509,17 +520,7 @@ public UniquePersonList getBlacklistedPersons() {
 }
 ----
 
-Although this seems inefficient, it supports syncing of the various lists well. Meaning, any changes that occurred in main list will be reflected in blacklist as well. For example, if a person is deleted from the main list, he will also be deleted from the blacklist. This also works for various other commands. Thus, it is efficient in this aspect.
-
-The following sequence diagram, Figure 3.5.1, shows further details of the interaction between the user and various application components as a whole, when the blacklist command is executed:
-
-image::BlacklistCommandSequenceDiagram1.PNG[width="800"]
-Figure 3.5.1 : Sequence Diagram of How the Blacklist Command Works_
-
-The following sequence diagram, Figure 3.5.2, shows further details of the interaction between `Model` and 'Addressbook` to obtain the blacklist:
-
-image::BlacklistCommandSequenceDiagram2.png[width="800"]
-Figure 3.5.2 : Sequence Diagram of Model and Addressbook Interacts To Generate The Blacklist From All Persons_
+Although this way of implementation seems inefficient, it supports robust synchronisation among the various other lists. Meaning, any changes that occurred in masterlist will be reflected in the blacklist as well. For example, if a person is deleted from the masterlist, he will also be deleted from the blacklist. Likewise for various other commands that changes the `ReadOnlyPerson` object. Thus, it is efficient in this aspect. Moreover, this implementation sets the groundwork for future implementations of various other lists.
```

Comment:
you can take out the entire sentence that starts with "meaning, "

Reply:
Noted.